app/controllers/items/new.js

```diff
@@ -169,6 +170,7 @@ export default Ember.Controller.extend({
       length: _this.get("length"),
```

Comment:
If anyone gets a change, we should remove all those useless `_this` references :)
we can use lambdas or `bind` to handle scope.

just a side note for future reference 👍

Reply:
I've already did this at many places in this file :)

Reply:
Its done already @patrixr .